### ISSUE TITLE
Edit Questionset: [archive/metadata-and-referencing/structure-granularity-and-referencing-pids]

### ISSUE THREAD

**Issue by smohex** [FDM-UARuhr/rdmo-catalog-uaruhr#43]
**Describe the request**  

In the help text the link to the online tutorial of the project DASISH leads to an error page

**Propose what exactly should be changed**  
The link should be replaced or removed

**Original setting or content**   
en: ... an <a href="http://training.dasish.eu/training/3/" target="_blank">online tutorial</a> created by the DASISH project or ...
de: ... in einem vom Projekt DASISH <a href="http://training.dasish.eu/training/3/" target="_blank">Online-Tutorial</a> oder...

**Proposed setting or content**   
Since I don't know which online tutorial should be linked, I can't suggest a correct link here. The project page dasish.eu seems to no longer exist

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  

- [ ] ~questionset~
- [x] help text (en and ger)

**Comment by smohex** [comment]
One more word should be added to the help text:
"Die PID bleibt gleich, auch wenn der Speicherort des Objektes sich **ändert**.

**Comment by ClaraRosalie** [comment]
remove link
